_ont/ont-nokia-g-010s-a.md

```diff
@@ -287,6 +287,7 @@ omciMgr
 - [Usage GPON module SFP in Spain](https://forum.mikrotik.com/viewtopic.php?t=116364&start=300)
 - [Bypassing the HH3K up to 2.5Gbps using a BCM57810S NIC](https://www.dslreports.com/forum/r32230041-Internet-Bypassing-the-HH3K-up-to-2-5Gbps-using-a-BCM57810S-NIC)
 - [Nokia G-010S-A Pin 6 Iusse - Rsaxvc.net](https://rsaxvc.net/blog/2020/8/15/Nokia_G-010S-A_Pin_6_Issue.html)
+- [Might require solder mod in order to work with BCM57810S NIC](/_sfp_cage/broadcom-57810s.md#compatibility)
```

Comment:
Link not working, the PR will now build the preview site so you can check it :)

Reply:
fixed